### ISSUE TITLE
Open a pre-populated email similarly to a QGIS action

### ISSUE THREAD

**Issue by pietroblu** [g3w-suite/g3w-client#588]
### Checklist

- [X] I've searched through the [current issues](https://github.com/g3w-suite/g3w-client/issues) to make sure this feature hasn't been requested already.

### Motivation

I need to allow a web map user to automate the creation of an email that contains the values of some fields of a vector layer, and then send it with the email client.

### Suggested solution

In QGIS it is possible to create an action to allow a user to create a pre-filled email that contains the values of some fields of the vector layer. This will open the email client and create an email ready to be sent to a specific sender.
When loading the QGIS project in G3W-Suite the actions don't seem to be imported, is there a way to obtain the same result?

**Comment by leolami** [comment]
Hi @pietroblu 

The actions are not yet managed by the suite
Except for static links

The aspect you are interested in (click on button and email pre-configuration) is manageable but also in this case with static contents

You can see an example on the LINK field of this geometry

https://v37.g3wsuite.it/it/map/demo-37/?zoom_to_fid=buildings_2f43dc1d_6725_42d2_a09b_dd446220104a%7C58

**Comment by leolami** [comment]
Consider that the content of this field can be updated automatically, in an editing session, by varying the content of another field (e.g. the one containing the email address associated with the feature) using the default values ​​on QGIS and therefore then into the suite

If it's not clear let me know

**Comment by pietroblu** [comment]
Hi @leolami 
thank you for your suggestions.

Indeed I tried to include in a field this demo HTML code 
`<a href="mailto:info@mysite.com?subject=Segnalazione oggetto &body=Segnalazione riguardo l'oggetto XX"></a>`
but stupidly I forgot to include some text between the tags <a href..></a> so obviously there was nothing to click on :)

Thanks again.

**Comment by leolami** [comment]
If you are using G3W-SUITE for a public project we would love to hear about the case study

If possible, share the result with us, we can also give you advice on how to make the most of the suite's features

Thank you
Leonardo

**Comment by pietroblu** [comment]
Hi @leolami 
thank you for your help, here is the deployment with some demo maps http://webgis.terralogic.it:8080/

I had already deployed v3.6.4 of the suite, but not having found a way to update to v3.7.x I decided to reinitialize the virtual machine and do a new deployment.

One thing I noticed is that with Brave browser the search bar, zoom, identify, pan, etc. tools are missing, as visible in the first screenshot, while they are shown if you browse with an incognito window or in the Edge browser (see second screenshot). I don't understand where the problem could be.

Thanks for any feedback

![image](https://github.com/g3w-suite/g3w-client/assets/3209793/077e13af-483e-4071-9623-a2106fd0d834)

![image](https://github.com/g3w-suite/g3w-client/assets/3209793/56290d15-100a-4270-be36-2d85a529dafa)
